src/plugins/vis_augmenter/public/view_events_flyout/components/styles.scss

```diff
@@ -0,0 +1,60 @@
+$vis-description-width: 200px;
+$event-vis-height: 55px;
+$timeline-panel-height: 100px;
+$content-padding-top: 110px; // Padding needed within view events flyout content to sit comfortably below flyout header
+$date-range-height: 45px; // Static height we want for the date range picker component
+$error-icon-padding-right: -8px; // This is so the error icon is aligned consistent with the event count icons
```

Comment:
Do the flyout and date-picker change sizes with text being enlarged? If yes, we should use `em` or `rem` here.